openmls/src/framing/mod.rs

```diff
@@ -39,15 +39,68 @@ pub use validation::*;
 mod test_framing;
 
 /// Wire format of MLS messages.
+///
+/// # MLS Presentation Language
+///
+/// ```c
+/// // draft-ietf-mls-protocol-16
+///
+/// enum {
+///   reserved(0),
+///   mls_plaintext(1),
+///   mls_ciphertext(2),
+///   mls_welcome(3),
+///   mls_group_info(4),
+///   mls_key_package(5),
+///   (255)
+/// } WireFormat;
+/// ```
 #[derive(
     PartialEq, Eq, Clone, Copy, Debug, Serialize, Deserialize, TlsDeserialize, TlsSerialize, TlsSize,
 )]
 #[repr(u8)]
 pub enum WireFormat {
-    /// Plaintext message
+    /// Reserved
+    Reserved = 0,
```

Comment:
We don't enumerate the reserved values, instead we rely on the decoder to fail if they are ever encountered

Reply:
Ok 👍🏻 I'll put a note in the comment and remove the `Reserved` variant.